BootloaderCorePkg/Library/MpInitLib/Ia32/MpInitLibInternal.h

```diff
@@ -37,6 +37,9 @@
 #define   SMM_BASE_GAP             0x1000
 #define   SMM_BASE_MIN_SIZE        0x10000
 
+#define   PLD_TO_LDR_SMM_SIG       0x11223344
```

Comment:
Can we use a meaningful signature ?   :)

Reply:
Changed the signature in the new patch.